spec/datadog/appsec/configuration/settings_spec.rb

```diff
@@ -77,6 +77,30 @@ def patcher
       end
     end
 
+    describe '#rasp_enabled' do
+      subject(:rasp_enabled) { settings.appsec.rasp_enabled }
+
+      context 'when DD_APPSEC_RASP_ENABLED' do
+        around do |example|
+          ClimateControl.modify('DD_APPSEC_RASP_ENABLED' => rasp_enabled_env_var) do
+            example.run
+          end
+        end
+
+        context 'is not defined' do
+          let(:rasp_enabled_env_var) { nil }
+
+          it { is_expected.to eq true }
+        end
+
+        context 'is defined' do
+          let(:rasp_enabled_env_var) { 'false' }
+
+          it { is_expected.to eq(false) }
```

Comment:
TBH for me it's harder to read what this `is_expected` referred to, I have to scroll to the very top definition. What if instead we do this

```suggestion
          it { expect(settings.appsec.rasp_enabled).to eq(false) }
```